sdcm/utils/replication_strategy_utils.py

```diff
@@ -95,3 +96,17 @@ def __call__(self, **keyspaces: ReplicationStrategy) -> None:
         for keyspace, strategy in keyspaces.items():
             self._preserve_replication_strategy(keyspace)
             strategy.apply(self.node, keyspace)
+
+
+@dataclass
+class TabletsConfiguration:
```

Comment:
Not sure this one directly related to replication

probably related to keyspaces, and we don't have a specific module for those yet

Reply:
May we leave it without change until maintainers team implements it?
Or I can do it by myself, just give me an idea where to place this new module (to prevent further questions) and I'll put this class there.

Reply:
it is kind-of-related, I don't know better place for now.

Reply:
we probably should have something similar to `sdcm/utils/sstable/sstable_utils.py` like `sdcm/utils/keyspace/keyspace_utils.py`

Reply:
One more approach, I can add a new module called `tablets` into `sdcm/utils/` and move this code there. Later, anyone who continue to work with tablets may extend or rework it depending on his needs.
@soyacz @fruch What do you think about it?

Reply:
I'm o.k. with that

Reply:
okay, will rework then

Reply:
Fixed and retested here:
https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/manager-master/job/ubuntu22-vnodes-tablets-cluster-test/13/